backend/app/api/routes/images.py

```
@@ -0,0 +1,176 @@
+import uuid
+from typing import Any, List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)